src/albums.go

```diff
@@ -0,0 +1,89 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"regexp"
+)
+
+// TODO : next feature
+// *  Add possibility to reload a directory or add new directory at the runtime
```

Comment:
This will be done in the second iteration, I think it's very important but I try to add value first then improve.